src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
```

Comment:
Violation: FINDBUGS Rule: Optional.of(EQ_ALWAYS_TRUE) Severity: ERROR
File: se/bjurr/violations/lib/example/CopyOfMyClass.java (se.bjurr.violations.lib.example.CopyOfMyClass)
equals method always returns true


  <p> This class defines an equals method that always returns true. This is imaginative, but not very smart.
Plus, it means that the equals method is not symmetric.
</p>


<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>